parm/stats/stat_analysis.yaml.j2

```diff
@@ -0,0 +1,3 @@
+time window:
```

Comment:
I think we are going to need one of these for each component, let's discuss later this morning